subsys/bluetooth/shell/iso.c

```diff
@@ -539,6 +605,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(iso_cmds,
 	SHELL_CMD_ARG(disconnect, NULL, "Disconnect ISO Channel",
 		      cmd_disconnect, 1, 0),
 #endif /* CONFIG_BT_ISO_UNICAST */
+#if defined(CONFIG_BT_ISO_BROADCASTER) || defined(CONFIG_BT_ISO_UNICAST)
+	SHELL_CMD_ARG(tx_sync_read, NULL, "[bis/cis]",
+		      cmd_tx_sync_read, 1, 1),
```

Comment:
I suggest defining separate shell commands for bis and cis. Either just `tx_sync_read_bis` and `tx_sync_read_cis`, or with a subcmd set. I think the code will end up simpler.

Reply:
True, the code would end up simpler. Separate commands would probably be the easiest, as I could organize them under the existing "#if CIS", and "#if BIS"categories

Reply:
Would also the match the split implement of `bt_iso_chan_send` commands :)